api/steps/step-id-4-6.json

```diff
@@ -0,0 +1,18 @@
+{ "id": "step-id-4-6",
```

Comment:
Minor: The naming I was using for these was `step-id-[taskId]-[stepIndex]` to ensure uniqueness. While `step-id-4-6` is unique, it might be confusing if someone adds another step to task 4